### DISCUSSION TITLE
修改了wiki工作区的配置是不是应该重启应用？

### DISCUSSION THREAD

**Discussion by dongrentianyu** [tiddly-gittly/TidGi-Desktop#540] [Q&A] [answered]
总感觉修改了wiki里的配置，应该要重启应用。因为不重启应用修改后的配置就不会表现出来。

但修改后又没有弹出重启应用的按钮。不像在总的设置里修改了创建者名称时，会弹出重启按钮。而且如果不点，也会在三秒还是五秒内自动重启。

因为不确定，所以先来讨论这边问一下。

**Comment by linonetwo** [comment, marked as answer]
新版已加上，只是没发布而已，再多修几个再发布。

**Reply by linonetwo** [reply]
我发个预发布版。

**Reply by linonetwo** [reply]
https://github.com/tiddly-gittly/TidGi-Desktop/releases/tag/v0.9.4-prerelease
